google-cloud-pubsub/test/google/cloud/pubsub_test.rb

```diff
@@ -564,4 +564,4 @@ def creds.is_a? target
       end
     end
   end
-end
+end
```

Comment:
Nit: Don't delete the newline at the end of the file. (Also appears in some other files in this PR)